rules/manager.go

```diff
@@ -121,8 +152,9 @@ type ManagerOptions struct {
 	ConcurrentEvalsEnabled    bool
 	RuleConcurrencyController RuleConcurrencyController
 	RuleDependencyController  RuleDependencyController
-
-	Metrics *Metrics
+	Metrics                   *Metrics
+	AlertStore                AlertStore
+	AlertStoreFunc            AlertStateStoreFunc
```

Comment:
Why do we need two things here?  
It seems one is called to set alerts and one to get them, but that is oddly asymmetric.

Reply:
Good point, AlertStoreFunc can be removed. I will update.